pulp_container/app/tasks/sync_stages.py

```diff
@@ -129,10 +134,19 @@ async def run(self):
                 tag_dc = DeclarativeContent(Tag(name=tag_name))
 
                 content_data = json.loads(raw_data)
-                media_type = content_data.get("mediaType")
+                media_type = content_data.get("mediaType") or dl_res.headers["content-type"]
+                try:
+                    schema = determine_schema(media_type)
+                except ValueError:
+                    raise RuntimeError(
+                        f"Requested a validation schema for an unknown media type: '{media_type}'"
```

Comment:
I'd also raise here ManifestInvalid error

Reply:
Can we re-use validate_manifest in both sync and push codepaths? expose it in utils.